### DISCUSSION TITLE
Bitwarden CLI send File

### DISCUSSION THREAD

**Discussion by m0rph2k** [dani-garcia/vaultwarden#4431] [Q&A]
Hello

I want to make a file upload via the CLI I do something wrong with this command "bw.exe send -f ./Test.zip" ?
I always get this error. A text upload works

400 Bad Request
The request could not be understood by the server due to malformed syntax.

image: vaultwarden/server:1.30.5
CLI version 18.5.0.0 (bw-windows-2024.2.1)

The latest version that works for me with Vaultwarden is bw-windows-2023.4.0
The current versions (bw-windows-2024.2.1) work on the Bitwarden Enterprise Server

**Comment by BlackDex** [comment]
I'm afraid this has nothing to do with Vaultwarden.

See https://github.com/bitwarden/clients/issues/5876
While that issue is closed, it has links and mentions to others which are still open.

Using the cli doesn't work on there self hosted instances.
